ui/css/perf.css

```diff
@@ -363,7 +363,10 @@ th {
 }
 
 .alert-summary-dropdown-element {
-  padding: 9px 5px 37px 95px;
+  display: flex;
```

Comment:
Could we replace this raw CSS with Bootstrap/reactstrap utilities?
For example, we have support for [flex components](https://getbootstrap.com/docs/4.0/utilities/flex/) & [margin/padding](https://getbootstrap.com/docs/4.0/utilities/spacing/).